packages/api/src/rpc/cennzx.ts

```diff
@@ -1,49 +1,63 @@
-import {RpcMethodOpt} from '@polkadot/jsonrpc/types';
-
-import createMethod from '@polkadot/jsonrpc/create/method';
-import createParam from '@polkadot/jsonrpc/create/param';
-
-const buyPrice: RpcMethodOpt = {
-  description: 'Retrieves the spot exchange buy price',
-  isOptional: true,
-  params: [
-    createParam('AssetToBuy', 'AssetId'),
-    createParam('Amount', 'Balance'),
-    createParam('AssetToPay', 'AssetId'),
-  ],
-  type: 'Balance',
+const cennzxRpc = {
+  buyPrice: {
+    description: 'Retrieves the spot exchange buy price',
+    params: [
+      {
+        name: 'AssetToBuy',
+        type: 'AssetId',
+      },
+      {
+        name: 'Amount',
+        type: 'Balance',
+      },
```

Comment:
It seems `AssetToPay` attribute is missing here;

Also there is a PR opening addressed relative refactoring where it shares the same approach like what are up to, for more details, please check out here - https://github.com/cennznet/api.js/pull/102/files

Relative refactoring may need to be conducted in `cennznet-ui`, and an opening PR addressed it as well, for more details, please check out here - https://github.com/cennznet/ui/pull/30/files